sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/src/AuthenticationEventResource.Designer.cs

```diff
@@ -269,7 +269,7 @@ internal class AuthenticationEventResource {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to Actions can not contain null items.
+        ///   Looks up a localized string similar to Actions can not contain null items..
```

Comment:
Double dot

Reply:
This file is auto-generated and updated. The actual exception strings are in the resx class.

Reply:
yeah the value looks good `Actions can not contain null items.`